templates.tpl

```diff
@@ -0,0 +1,136 @@
+  },
+  "description": "PBX Plus is an advanced communications system and IVR for your business needs.\nhttps://infinity.500apps.com/ PBXPlus",
+  "containerContexts": [
+    "WEB"
+  ]
+}
+
+
+___TEMPLATE_PARAMETERS___
+
+[
+  {
+    "type": "TEXT",
+    "name": "api_key",
+    "displayName": " PBXPlus Key",
+    "simpleValueType": true,
+    "valueValidators": [
+      {
+        "type": "NON_EMPTY"
+      }
+    ],
+    "help": "Please copy the  PBXPlus from App -\u003eCreate a form -\u003e Embed. https://infinity.500apps.com/ PBXPlus"
+  }
+]
+
+
+___SANDBOXED_JS_FOR_WEB_TEMPLATE___
+
+const log = require('logToConsole');
+const injectScript = require('injectScript');
+const encodeUriComponent = require('encodeUriComponent');
+const api_key = encodeUriComponent(data.api_key);
+const trackingUrl='https://gtm.500apps.com/gtm/ PBXPlus/'+api_key+'.js';
```

Comment:
why there is space


<img width="283" alt="Screenshot 2021-12-08 at 8 06 35 AM" src="https://user-images.githubusercontent.com/66065056/145137709-896b1d5b-5c6d-46f0-9e63-a24b79b1e97e.png">
no formatter

Reply:
resolved